woocommerce/woocommerce-android
# Center more menu items text

Closes: #9239


### Description

Centers the text on the more menu buttons. Before it was alleged to the top

### Testing instructions

* Open the "more menu" screen
* Notice centered text

### Images/gif


| Before | After |
| ------------- | ------------- |
| <img width="292" alt="image" src="https://github.com/woocommerce/woocommerce-android/assets/4923871/5a011259-a8f6-4ab1-853b-8c31c5fffbc0" />  | <img width="292" alt="image" src="https://github.com/woocommerce/woocommerce-android/assets/4923871/93ecfa43-1287-44a7-bbfb-9256dc61e4d4"/> |
